### PR TITLE
Feature/ Implement delete modal

### PR DESCRIPTION
**Pull request by JennyDoJD**
Result:
image

Ticket: [Implement Delete feature](https://trello.com/b/8u43DOmI/javascript-training)

### CODE REVIEW

File: practice/src/index.html [hunk 1]
```diff
@@ -135,6 +135,20 @@ <h1 class="main-title">Products Management</h1>
           <!-- Card Products-->
           <div id="product-list" class="card-product-list"></div>
         </main>
+        <!-- Delete Modal -->
+        <div class="delete-modal">
+          <div class="delete-model-content">
```
Comment (huyencao-agilityio) [inline practice/src/index.html]: Typo `Model`

File: practice/src/index.html [hunk 2]
```diff
@@ -135,6 +135,20 @@ <h1 class="main-title">Products Management</h1>
...
+        <!-- Delete Modal -->
+        <div class="delete-modal">
+          <div class="delete-modal-content">
+            <p class="delete-modal-text">Are you sure you want to delete this food?</p>
+            <div class="delete-modal-button">
+              <button class="btn btn-primary btn-confirm" type="reset">
+                Cancel
+              </button>
+              <button class="btn btn-secondary btn-confirm" type="submit">
+                Save
```
Comment (ghost) [inline practice/src/index.html]: @JennyDoJD It should be Yes or Ok

File: practice/src/index.html [hunk 3]
```diff
@@ -135,6 +135,20 @@ <h1 class="main-title">Products Management</h1>
...
         </main>
+        <!-- Delete Modal -->
+        <div class="delete-modal">
+          <div class="delete-modal-content">
+            <p class="delete-modal-text">Are you sure you want to delete this food?</p>
+            <div class="delete-modal-button">
+              <button class="btn btn-primary btn-confirm" type="reset">
+                Cancel
+              </button>
+              <button class="btn btn-secondary btn-confirm" type="submit">
```
Comment (ghost) [inline practice/src/index.html]: why should we need type="submit"?

**Review by ghost — CHANGES_REQUESTED** [review]

File: practice/src/index.html (hunk 3 above)
Comment (JennyDoJD) [reply practice/src/index.html]: it's the type="button"

**Review by huyencao-agilityio — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**